server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -935,21 +937,24 @@ private ClusterMetadataManifest fetchRemoteClusterMetadataManifest(String cluste
 
     private ChecksumBlobStoreFormat<ClusterMetadataManifest> getClusterMetadataManifestBlobStoreFormat(String fileName) {
         long codecVersion = getManifestCodecVersion(fileName);
-        if (codecVersion == ClusterMetadataManifest.CODEC_V0) {
+        if (codecVersion == ClusterMetadataManifest.CODEC_V1) {
```

Comment:
initialize CODEC_CURRENT  = CODEC_V1

Reply:
We already have `MANIFEST_CURRENT_CODEC_VERSION`, using that over here.